.github/workflows/typecheck.yml

```diff
@@ -23,8 +23,6 @@ jobs:
           cache: 'pnpm'
       - name: Install dependencies
         run: pnpm install
-      - name: Build docs for generated files
```

Comment:
I added it back to command in package.json so the typecheck also doesn't break when nothing is build